components/ILIAS/Forum/classes/class.ilObjForumGUI.php

```diff
@@ -2448,12 +2445,10 @@ private function initReplyEditForm(): void
                 ($isDraft && $this->objCurrentPost->getDepth() >= 2)
             )
         );
-        if ($quotingAllowed) {
-            $oPostGUI->addPlugin('ilfrmquote');
-            $oPostGUI->addButton('ilFrmQuoteAjaxCall');
-        }
 
         $oPostGUI->removePlugin('advlink');
+        $oPostGUI->removePlugin('ilimgupload');
```

Comment:
If https://github.com/ILIAS-eLearning/ILIAS/pull/8091 got merged, we don't need this anymore.